mmv1/products/securitycenterv2/OrganizationNotificationConfig.yaml

```diff
@@ -0,0 +1,130 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'OrganizationNotificationConfig'
+base_url: organizations/{{organization}}/locations/{{location}}/notificationConfigs
+self_link: '{{name}}'
+create_url: organizations/{{organization}}/locations/{{location}}/notificationConfigs?configId={{config_id}}
+update_verb: :PATCH
+update_mask: true
+description: |
+  A Cloud Security Command Center (Cloud SCC) notification configs. A
+  notification config is a Cloud SCC resource that contains the
+  configuration to send notifications for create/update events of
+  findings, assets and etc.
+  ~> **Note:** In order to use Cloud SCC resources, your organization must be enrolled
+  in [SCC Standard/Premium](https://cloud.google.com/security-command-center/docs/quickstart-security-command-center).
+  Without doing so, you may run into errors during resource creation.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/security-command-center/docs'
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v2/organizations.locations.notificationConfigs'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_v2_organization_notification_config_basic'
+    primary_resource_id: 'custom_organization_notification_config'
+    vars:
+      topic_name: 'my-topic'
+      config_id: 'my-config'
+    test_env_vars:
+      org_id: :ORG_ID
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  custom_import: templates/terraform/custom_import/scc_source_self_link_as_name_set_organization.go.erb
+  post_create: templates/terraform/post_create/set_computed_name.erb
+parameters:
+  - !ruby/object:Api::Type::String
+    name: organization
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The organization whose Cloud Security Command Center the Notification
+      Config lives in.
+  - !ruby/object:Api::Type::String
+    name: configId
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      This must be unique within the organization.
+  - !ruby/object:Api::Type::String
+    name: location
+    required: false
+    immutable: true
+    url_param_only: true
+    default_value: global
```

Comment:
Could you please confirm that you want this to be the long-term default for this resource? Asking because changing defaults is a breaking change, so it would be difficult to change this later

Reply:
This is mentioned in the requirement by client. To keep the default_value = global
https://docs.google.com/document/d/1b97vduAO9BPiTTFa-muB9c9ACuDFlAOZW745PrkohXM/edit?tab=t.0